src/DynamoCoreWpf/Controls/DynamoTextBox.cs

```diff
@@ -390,26 +390,20 @@ public void SetDataContext(object dataContext, bool closeImmediately = false)
             // If Dynamo window is not active, we should not show as well as hide tooltip or do any other staff.
             if (mainDynamoWindow == null || !mainDynamoWindow.IsActive) return;
 
-            if (dataContext == null)
+            if (closeImmediately)
             {
-                if (closeImmediately)
-                {
-                    CloseLibraryToolTipPopup(null, null);
-                    return;
-                }
-                showTimer.Stop();
-                dispatcherTimer.Start();
+                CloseLibraryToolTipPopup();
                 return;
             }
-            dispatcherTimer.Stop();
-            nextDataContext = dataContext;
-            showTimer.Start();
+
+            toolTipTimer.Start(dataContext, 150);
         }
 
-        private void OpenLibraryToolTipPopup(object sender, EventArgs e)
+        private void OpenCloseLibraryToolTipPopup(object dataContext)
         {
-            this.DataContext = nextDataContext;
-            IsOpen = true;
+            IsOpen = dataContext != null || this.IsMouseOver;
+            if (dataContext != null)
+                this.DataContext = dataContext;
```

Comment:
Oh, here i'm sure!
Example(let's remove if-statement):
1. User moves mouse on member. (this.DataContext = dataContext;)
2. Tooltip is open.
3. User moves mouse on tooltip, in this case dataContext==null, since every time mouse moves out of member button tooltip should be hidden. (this.DataContext = dataContext;) <= As result tooltip is open, but empty.
We should check if datacontext is null, so that we can show tooltip, when mouse is over.